packages/failurechecker/src/failurechecker.ts

```diff
@@ -23,12 +23,10 @@ import {IssuesListForRepoResponseData} from '@octokit/types';
 type OctokitType = InstanceType<typeof Octokit>;
 
 import {logger} from 'gcf-utils';
+
 // labels indicative of the fact that a release has not completed yet.
-const RELEASE_LABELS = [
-  'autorelease: pending',
-  'autorelease: tagged',
-  'autorelease: failed',
-];
+const RELEASE_LABELS = ['autorelease: pending', 'autorelease: failed'];
+const RELEASE_TYPE_NO_PUBLISH = ['go-yoshi'];
```

Comment:
I think this solution makes sense for right now. But, if this number expands, how do we account for the scale? Could we install it on a repo-level (vs. an org level)?